src/components/message/message-card/MessageCard.tsx

```diff
@@ -193,7 +194,64 @@ function MessageCardBase({ message, viewType, setViewType }: Props) {
 		messagesStore.setHoveredMessage(null);
 	};
 
+	const renderMessageInfo = () => {
```

Comment:
I think we can combine renderMessageInfo and renderInlineMessageInfo into one method that takes MessageViewType